client/src/pages/Seeds.tsx

```diff
@@ -0,0 +1,55 @@
+import {A1, A2, A3} from "../assets/images"
+
+export const resultSeed = [
+  {
+    name: "Mom's",
+    distance: 3.4 + "mi",
+    keyword: "vegan",
+    price: "$$",
+  },
+  {
+    name: "Bread Basket",
+    distance: 1.1 + "mi",
+    keyword: "raw",
+    price: "$$$",
+  },
+  {
+    name: "Chotchkie's",
+    distance: 13.2 + "mi",
+    keyword: "american",
+    price: "$",
+  },
+  {
+    name: "Ramen Shop",
+    distance: 3 + "mi",
+    keyword: "japanese",
+    price: "$$",
+  },
+];
+
+export const reviewSeed = [
```

Comment:
What is the difference between this `reviewSeed` and the `reviewSeed` in `client/src/pages/Reviews/reviewSeed.tsx`?